### PR TITLE
[MIG][15.0] l10n_fr_siret

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: l10n_fr_siret/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        inverse="_inverse_siret",
+        store=True,
+        readonly=False,
+        help="The SIRET number is the official identity number of this "
+        "company's office in France. It is composed of the 9 digits "
+        "of the SIREN number and the 5 digits of the NIC number, ie. "
+        "14 digits.",
+    )
+    company_registry = fields.Char(
+        size=64,
```
Comment (legalsylvain) [inline l10n_fr_siret/models/res_partner.py]: Does it makes sense ? I think we could remove that limitation.

**Review by legalsylvain — APPROVED** [review]
Diff review. A minor comment.
Lgtm otherwise. Thanks !
Do you know why there a lot of empty translation files ?
Maybe we could remoce them ? 
Regards and happy new year.

File: l10n_fr_siret/models/res_partner.py (hunk 1 above)
Comment (alexis-via) [reply l10n_fr_siret/models/res_partner.py]: ok, I removed it

**Pull request merged**